src/entities/offerings.ts

```diff
@@ -96,6 +151,23 @@ export interface Product {
    * The offering ID used to obtain this product.
    */
   readonly presentedOfferingIdentifier: string;
+
+  /**
+   * The default option id to be used to subscribe to this product.
+   * Null if no subscription options are available.
```

Comment:
Hmm I think this should never happen except for in-app products once we add support for them right? I think we should mention that here.